modules/index-service/src/main/java/org/opencastproject/index/service/message/SchedulerMessageReceiverImpl.java

```diff
@@ -78,92 +78,77 @@ private void executeSingle(final String mediaPackageId, final SchedulerItem sche
           if (dc != null)
             EventIndexUtils.updateEvent(event, dc);
         } catch (SearchIndexException e) {
-          logger.error("Error retrieving the recording event from the search index: {}", getStackTrace(e));
+          logger.error("Error retrieving the recording event from the search index", e);
           return;
```

Comment:
@lkiesow  since you have the mpId already, it'd be very helpful to add it to the error log so there's less digging for context when this log surfaces.